fixtures/dev-env/tests/index.test.ts

```diff
@@ -649,3 +649,42 @@ describe("startDevWorker: ProxyController", () => {
 		await expect(res.text()).resolves.toBe("UserWorker:3");
 	});
 });
+
+describe("regressions", () => {
+	// https://github.com/cloudflare/workers-sdk/issues/5095
+	test("#5095 requests where body (of >= 2 bytes) is not drained sometimes fail", async () => {
```

Comment:
```suggestion
	test("requests where body (of >= 2 bytes) is not drained sometimes fail", async () => {
```
Could this description describe what the test is testing? Is it checking that the regression is fixed, or demonstrating a code path that fails?